Validation/RecoTrack/python/MTVHistoProducerAlgoForTrackerBlock_cfi.py

```diff
@@ -147,5 +149,20 @@ def _modifyForPhase1(pset):
     pset.nintEta = 60
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
 phase1Pixel.toModify(MTVHistoProducerAlgoForTrackerBlock, _modifyForPhase1)
+
+def _modifyForPhase2(pset):
+    pset.minEta = -4.5
+    pset.maxEta = 4.5
+    pset.nintEta = 90
+
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
-phase2_tracker.toModify(MTVHistoProducerAlgoForTrackerBlock, minEta=-4.5, maxEta=4.5, nintEta = 90)
+phase2_tracker.toModify(MTVHistoProducerAlgoForTrackerBlock, _modifyForPhase2)
+
+def _modifyForPhase2wMTD(pset):
+    pset.minEta = -4.5
+    pset.maxEta = 4.5
+    pset.nintEta = 90
+    doMTDPlots = cms.untracked.bool(True)
```

Comment:
Isn't the binning of eta more a property of the tracker than the MTD? If that is the case, it is redundant to repeat the eta binning customization here (AFAIK all phase2 eras with `phase2_timing_layer` include `phase2_tracker`).

Also, it is preferred to avoid specifying the type (`cms.untracked.bool`) if not necessary (here it is not).